src/helpers/pivot/pivot_helpers.ts

```diff
@@ -184,3 +234,11 @@ export function extractFormulaIdFromToken(tokenAtCursor: EnrichedToken) {
   }
   return idAst.value;
 }
+
+export function toDomainArgs(domainStr: string[]) {
+  const domain: DomainArg[] = [];
+  for (let i = 0; i < domainStr.length - 1; i += 2) {
+    domain.push({ field: domainStr[i], value: domainStr[i + 1] });
```

Comment:
maybe it could check the number of elements in the array is pair. Otherwise we might end up with undefined values and Typescript won't tell us.